build_tools/compute_rocm_package_version.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+# Copyright Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: MIT
+
+"""Computes a ROCm package version with an appropriate suffix for a given release type.
+
+Also writes to 'version' in GITHUB_OUTPUT if running under GitHub Actions.
+"""
+
+import argparse
+from datetime import datetime
+from pathlib import Path
+import json
+import os
+import subprocess
+import sys
+
+from github_actions.github_actions_utils import *
+
+THIS_SCRIPT_DIR = Path(__file__).resolve().parent
+THEROCK_DIR = THIS_SCRIPT_DIR.parent
+
+
+def _log(*args, **kwargs):
+    print(*args, **kwargs)
+    sys.stdout.flush()
+
+
+def load_rocm_version() -> str:
+    """Loads the rocm-version from the repository's version.json file."""
+    version_file = THEROCK_DIR / "version.json"
+    _log(f"Loading version from file '{version_file.resolve()}'")
+    with open(version_file, "rt") as f:
+        loaded_file = json.load(f)
+        return loaded_file["rocm-version"]
+
+
+def get_git_sha():
+    """Gets the current git SHA, either from GITHUB_SHA or running git commands."""
+
+    # https://docs.github.com/en/actions/reference/workflows-and-actions/variables
```

Comment:
maybe a bit more clarification
```suggestion
    # Default GitHub environment variable, info: https://docs.github.com/en/actions/reference/workflows-and-actions/variables
```